src/hotspot/share/jvmci/jvmci_globals.cpp

```diff
@@ -100,7 +100,7 @@ bool JVMCIGlobals::check_jvmci_flags_are_consistent() {
     }
     if (BootstrapJVMCI && (TieredStopAtLevel < CompLevel_full_optimization)) {
       jio_fprintf(defaultStream::error_stream(),
-          "-XX:+BootstrapJVMCI is not compatible with -XX:TieredStopAtLevel=%d\n", TieredStopAtLevel);
+          "-XX:+BootstrapJVMCI is not compatible with -XX:TieredStopAtLevel=" INTX_FORMAT "\n", TieredStopAtLevel);
```

Comment:
This change required changes to 18 Java files and then many test failures in mach5 tiers tests. So, I removed the changes to avoid diverging from this PR.